curvallis/run.py

```diff
@@ -361,36 +361,41 @@ def button_release_callback(self, event):
     def key_press_callback(self, event):
         """ Whenever a key is pressed
         """
-        print ('Pressed %s' % event.key)
-        if self._quit_pending:
-            if event.key == 'q':
-                print ('Quitting.')
+        print ('Pressed %s' % event.key) #Display name of pressed key
+        if self._quit_pending:                  #If a quit is pending
+            if event.key == 'q':                #and 'q' is pressed
+                print ('Quitting.')             #quit
                 exit(0)
-            else:
-                self._quit_pending = False
+            else:                               #Else:
+                self._quit_pending = False      #cancle quit pending
                 print ('Quit cancelled.')
-        elif event.key == 'q':
+        elif event.key == 'q':                  #If 'q' pressed
+                                                #Display pending quit
             print ('Quit requested.  Press q again to quit, any other key to cancel.')
             self._quit_pending = True
-        elif event.key == 'm':
-            self._print_keymap()
+        elif event.key == 'm':                  #If 'm' pressed
+            self._print_keymap()                #Display key mapping
         elif event.inaxes:
-            if event.key == 'c':
-                self._regions.plot_curves()
+            if event.key == 'c':                #If 'c' pressed         (not in key mapping)
+                self._regions.plot_curves()     #???
                 self._set_xlim_ylim()
                 self._canvas.draw()
```

Comment:
From off the top of my head, this key mapping doesn't seem to do any operation or change on the data, it simply re-plots/re-draws everything graphically. I would suppose that for this reason the key mapping isn't in the help menu. To be honest, should we just delete this key mapping? I know I personally don't use it, even in debugging. @sudo-Eric? @leek2?

Reply:
Figured it out. Pressing "c" just resets the view to the original or "home" view of all the data.